src/utils/utils.ts

```diff
@@ -73,3 +87,36 @@ export const calculateTimeRemaining = (startDate: Date) => {
     seconds: seconds % 60,
   };
 };
+
+/**
+ * Returns whether the given date is in the range of a given start and end date.
+ *
+ * @param targetDate - Optional. The date that is to be checked if it's in range of [startDate] and [endDate]. If not provided, the current date and time when the function is called will be used.
+ * @param startDate - A date, must be before [endDate].
+ * @param endDate - A date, must be after [startDate].
+ * @returns A boolean stating if the current date is greater than or equal to startDate and less than or equal to endDate.
+ */
+export const dateInRange = (
+  targetDate: Date = new Date(),
+  startDate: Date,
+  endDate: Date
```

Comment:
```suggestion
  startDate: Date,
  endDate: Date
  targetDate: Date = new Date(),
```
Put optional parameters after. Make sure to change the doc string order.